tests/strands/event_loop/test_event_loop.py

```diff
@@ -484,7 +521,7 @@ def test_event_loop_cycle_stop(
         tool_execution_handler=tool_execution_handler,
         request_state={"stop_event_loop": True},
     )
-    event = list(stream)[-1]
+    event = [event async for event in stream][-1]
```

Comment:
Can we have a helper to do this - presumedly we'll need to collect events a lot? (maybe not though)